docs/actions/view-action.md

```diff
@@ -0,0 +1,30 @@
+---
+title: View details of an Action
+intro: In this section you will learn how to find and understand details of a Devopness action
+links:
+    overview:
+    quickstart:
+    previous: /docs/actions
+    next:
+    guides:
+    related:
+        - /docs/actions
+    featured:
+---
+
+You can access details of all actions executed in an environment to which you have access.
+
+1. On Devopness, navigate to a project then select an environment
+1. On the upper-right corner, click `ACTIONS` to see a list of all actions created in the current environment
+1. In the list of actions, find the action you want to inspect and click the link in the `ACTION` column
+1. You will be redirected to the action details view
+1. In the action details view, the header displays:
+  - `Status`
+  - `Status reason`: a detailed explanation of why the action is in the current `Status`
+  - The resource type (e.g.: `Application`, `Server`, `Service`, ...) to which the action was created
+  - The operation being executed for that resource type (e.g.: `Deploy`, `Start`, `Stop`, ...)
+  - The name of the environment in which that action was created
+  - The start time
+  - The user, or automated webhook, who triggered the action
```

Comment:
Could you apply 4 spaces identation? 

This allows the website to render the unordered list without resetting the step numbering

```suggestion
    - `Status`
    - `Status reason`: a detailed explanation of why the action is in the current `Status`
    - The resource type (e.g.: `Application`, `Server`, `Service`, ...) to which the action was created
    - The operation being executed for that resource type (e.g.: `Deploy`, `Start`, `Stop`, ...)
    - The name of the environment in which that action was created
    - The start time
    - The user, or automated webhook, who triggered the action
```

### More info

BEFORE:

![image](https://github.com/user-attachments/assets/0dfe7e81-a2fe-463f-b223-c076d53be1ad)


AFTER:

![image](https://github.com/user-attachments/assets/acff7194-0cd0-4c97-af51-f579e7173e6c)